ui/pages/asset/components/asset-page.tsx

```diff
@@ -479,7 +479,7 @@ const AssetPage = ({
               {t('yourActivity')}
             </Text>
             {type === AssetType.native ? (
-              <TransactionList hideTokenTransactions hideNetworkFilter />
```

Comment:
nice fix!

Is this `hideTokenTransactions` still being used anywhere?

Reply:
If this is not used anywhere, I'm tempted to see if we can clean up this component and tests.